build.gradle

```diff
@@ -699,22 +696,6 @@ def getCheckedOutGitCommitHash() {
   grgit ? grgit.head().id.take(takeFromHash) : 'UNKNOWN'
 }
 
-task jacocoRootReport(type: JacocoReport) {
-  additionalSourceDirs.from = files(subprojects.sourceSets.main.allSource.srcDirs)
-  sourceDirectories.from = files(subprojects.sourceSets.main.allSource.srcDirs)
-  classDirectories.from = files(subprojects.sourceSets.main.output)
-  executionData.from = files(subprojects.jacocoTestReport.executionData)  //how to exclude some package/classes com.test.**
-  reports {
-    xml.enabled true
-    csv.enabled true
-    html.destination file("build/reports/jacocoHtml")
-  }
-  onlyIf = { true }
-  doFirst {
-    executionData = files(executionData.findAll { it.exists() })
-  }
-}
```

Comment:
Removed jacoco because we don't use it and it was causing tests to fail on java 15 for me.